app/models/claim.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: claims
+#
+#  id         :uuid             not null, primary key
+#  draft      :boolean          default(FALSE)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  school_id  :uuid             not null
+#
+# Indexes
+#
+#  index_claims_on_school_id  (school_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (school_id => schools.id)
+#
+class Claim < ApplicationRecord
+  belongs_to :school
```

Comment:
If you go with a non-standard name for the association, presumably we'll also want:

```suggestion
  belongs_to :school, inverse_of: :claim_applications
```

Although IMHO I think it'd probably be cleaner just to go with the conventional `:claims` name for this association.